src/Visualization/Python/Render1D.py

```diff
@@ -44,6 +44,10 @@ def parse_cmd_line():
                              help="List of all files if they do not "
                              "have same file prefix. You must include '.h5' "
                              "extension ")
+    parser.add_argument('--vol-file', type=str, required=True,
```

Comment:
could you rename this argument to `--subfile-name` to make `GenerateXdmf`? It'll be easier on folks if options are named consistently across files :)